template.yaml

```diff
@@ -166,6 +171,8 @@ Conditions:
 Globals:
   Function:
     Timeout: 60
+    PermissionsBoundary: !If [CreateDefaultBoundaryPolicy, !Ref DefaultPermissionBoundaryPolicy, !Ref PermissionBoundaryPolicyArn]
```

Comment:
Instead of creating and assigning a DefaultPermissionBoundaryPolicy, can we use `AWS::NoValue` to let CFN to not put `PermissionsBoundary`?